tests/testthat/test-backport_linter.R

```diff
@@ -64,3 +58,13 @@ test_that("backport_linter detects backwards-incompatibility", {
     backport_linter("3.0.0", except = c("numToBits", "R_user_dir"))
   )
 })
+
+test_that("backport_linter generates expected warnings", {
+  skip_if_not_installed("magrittr")
```

Comment:
we need `library(magrittr)` here though, no? since we're assuming magrittr is not attached, `%>%` won't be available.

options:

 1. just nest the calls
 2. use `` `%>%` <- magrittr::`%>%` ``

Reply:
Hmm, why doesn't this test fail? 
Is it because `{testthat}` itself relies on `{magrittr}`, and so the latter namespace is always loaded while running tests?

Reply:
I am no longer using magrittr pipes here.